galaxy/normalization/NmrNormalization_xml.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="normalization" name="Normalization" version="1.0.2">
+<tool id="normalization" name="Normalization" version="1.0.6">
```

Comment:
Why jump from 1.0.2 to 1.0.6?
Note that you should have a conflit one the version number when we will merge this PR https://github.com/workflow4metabolomics/normalization/pull/12
We can wait for your PR to merge it within the 1.0.3.
Can you change that?

Reply:
I don't understand: I made no change on the version number (on my file, this is still 1.0.2)

Reply:
I did it for you : https://github.com/workflow4metabolomics/normalization/pull/12/commits/0e3e89f82b863ac8859379c7c87125cd2fb2746b
I was afraid that you create a third PR if I ask you to do it 😝

Reply:
If you look at the Git tags, you will see that I've used the version number to tag. This version tag is used to build container in PhenoMeNal, and to identify them. If I want to use a more recent version of the tool inside the PhenoMeNal container, I have to upgrade the tool version. So I've made it several times in a row, because I was each time forgetting something. Sorry for this increase of the version number, but I'm afraid we can't go back to previous numbers.

Reply:
> If you look at the Git tags, you will see that I've used the version number to tag

I don't understand, sorry!
It's not a problem if I don't understand to go further on this PR 👍